src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -937,6 +937,7 @@ private static OpenApiMediaType CreateResponseContainer(string responseObjectSch
             {
                 Schema = new()
                 {
+                    Type = SCHEMA_OBJECT_TYPE,
```

Comment:
will anything else might help the Type property?

Reply:
We'd need an instance of Infragistics AppBuilder to test. But that is also whack-a-mole because different openapi client tooling sets different expectations on vague areas of the openapi spec. See referenced issues in PR description for examples of that point.